diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ff_cbest_1p32_125_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ff_cbest_1p32_125_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ff_cbest_1p32_125_nldm.lib	2026-07-31 19:37:27.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ff_cbest_1p32_125_nldm.lib	2026-08-21 14:20:00.714654677 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -591027,2 +591054,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -591152,2 +591180,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -591207,2 +591236,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ff_rcbest_1p08_125_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ff_rcbest_1p08_125_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ff_rcbest_1p08_125_nldm.lib	2026-07-31 19:37:28.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ff_rcbest_1p08_125_nldm.lib	2026-08-21 14:20:00.519657916 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -591027,2 +591054,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -591152,2 +591180,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -591207,2 +591236,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ff_rcbest_1p32_m40_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ff_rcbest_1p32_m40_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ff_rcbest_1p32_m40_nldm.lib	2026-07-31 19:37:29.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ff_rcbest_1p32_m40_nldm.lib	2026-08-21 14:20:00.172663682 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -591027,2 +591054,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -591152,2 +591180,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -591207,2 +591236,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ss_cworst_1p08_m40_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ss_cworst_1p08_m40_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ss_cworst_1p08_m40_nldm.lib	2026-07-31 19:37:30.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ss_cworst_1p08_m40_nldm.lib	2026-08-21 14:20:00.035665958 +0800
@@ -585885,2 +585885,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -586010,2 +586011,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -586061,2 +586063,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -586238,2 +586241,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -586363,2 +586367,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -586418,2 +586423,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -586570,2 +586576,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -586695,2 +586702,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -586746,2 +586754,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -586923,2 +586932,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587048,2 +587058,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587103,2 +587114,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -587289,2 +587301,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587414,2 +587427,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587469,2 +587483,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -587655,2 +587670,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587780,2 +587796,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587835,2 +587852,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588021,2 +588039,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588146,2 +588165,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588201,2 +588221,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588387,2 +588408,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588512,2 +588534,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588567,2 +588590,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588753,2 +588777,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588878,2 +588903,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588933,2 +588959,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589119,2 +589146,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589244,2 +589272,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589299,2 +589328,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ss_rcworst_1p08_125_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ss_rcworst_1p08_125_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ss_rcworst_1p08_125_nldm.lib	2026-07-31 19:37:31.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ss_rcworst_1p08_125_nldm.lib	2026-08-21 14:20:00.918651287 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -591027,2 +591054,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -591152,2 +591180,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -591207,2 +591236,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
Binary files ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/.ics55_LLSC_H7CH_ss_rcworst_1p08_125_nldm.lib.swp and IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/.ics55_LLSC_H7CH_ss_rcworst_1p08_125_nldm.lib.swp differ
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ss_rcworst_1p2_m40_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ss_rcworst_1p2_m40_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ss_rcworst_1p2_m40_nldm.lib	2026-07-31 19:37:32.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_ss_rcworst_1p2_m40_nldm.lib	2026-08-21 14:20:00.336660957 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -591027,2 +591054,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -591152,2 +591180,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -591207,2 +591236,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_typ_tt_1p2_25_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_typ_tt_1p2_25_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_typ_tt_1p2_25_nldm.lib	2026-07-31 19:37:33.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CH/liberty/ics55_LLSC_H7CH_typ_tt_1p2_25_nldm.lib	2026-08-21 14:19:59.901668189 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -591027,2 +591054,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -591152,2 +591180,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -591207,2 +591236,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ff_cbest_1p32_125_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ff_cbest_1p32_125_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ff_cbest_1p32_125_nldm.lib	2026-07-31 19:37:34.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ff_cbest_1p32_125_nldm.lib	2026-08-21 14:19:59.769670389 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -591027,2 +591054,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -591152,2 +591180,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -591207,2 +591236,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ff_rcbest_1p08_125_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ff_rcbest_1p08_125_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ff_rcbest_1p08_125_nldm.lib	2026-07-31 19:37:35.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ff_rcbest_1p08_125_nldm.lib	2026-08-21 14:19:59.507674754 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -591027,2 +591054,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -591152,2 +591180,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -591207,2 +591236,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ff_rcbest_1p32_m40_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ff_rcbest_1p32_m40_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ff_rcbest_1p32_m40_nldm.lib	2026-07-31 19:37:35.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ff_rcbest_1p32_m40_nldm.lib	2026-08-21 14:19:59.227679419 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -591027,2 +591054,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -591152,2 +591180,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -591207,2 +591236,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ss_cworst_1p08_m40_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ss_cworst_1p08_m40_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ss_cworst_1p08_m40_nldm.lib	2026-07-31 19:37:36.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ss_cworst_1p08_m40_nldm.lib	2026-08-21 14:19:59.362677170 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -591027,2 +591054,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -591152,2 +591180,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -591207,2 +591236,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ss_rcworst_1p08_125_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ss_rcworst_1p08_125_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ss_rcworst_1p08_125_nldm.lib	2026-07-31 19:37:37.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ss_rcworst_1p08_125_nldm.lib	2026-08-21 14:19:58.958683903 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -591027,2 +591054,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -591152,2 +591180,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -591207,2 +591236,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ss_rcworst_1p2_m40_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ss_rcworst_1p2_m40_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ss_rcworst_1p2_m40_nldm.lib	2026-07-31 19:37:38.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_ss_rcworst_1p2_m40_nldm.lib	2026-08-21 14:19:59.095681619 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -591027,2 +591054,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -591152,2 +591180,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -591207,2 +591236,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_typ_tt_1p2_25_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_typ_tt_1p2_25_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_typ_tt_1p2_25_nldm.lib	2026-07-31 19:37:39.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CL/liberty/ics55_LLSC_H7CL_typ_tt_1p2_25_nldm.lib	2026-08-21 14:19:59.637672588 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -591027,2 +591054,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -591152,2 +591180,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -591207,2 +591236,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ff_cbest_1p32_125_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ff_cbest_1p32_125_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ff_cbest_1p32_125_nldm.lib	2026-07-31 19:37:40.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ff_cbest_1p32_125_nldm.lib	2026-08-21 14:20:01.543640927 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -591027,2 +591054,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -591152,2 +591180,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -591207,2 +591236,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ff_rcbest_1p08_125_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ff_rcbest_1p08_125_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ff_rcbest_1p08_125_nldm.lib	2026-07-31 19:37:41.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ff_rcbest_1p08_125_nldm.lib	2026-08-21 14:20:02.551624250 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -591027,2 +591054,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -591152,2 +591180,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -591207,2 +591236,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ff_rcbest_1p32_m40_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ff_rcbest_1p32_m40_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ff_rcbest_1p32_m40_nldm.lib	2026-07-31 19:37:42.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ff_rcbest_1p32_m40_nldm.lib	2026-08-21 14:20:02.018633057 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -591027,2 +591054,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -591152,2 +591180,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -591207,2 +591236,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ss_cworst_1p08_m40_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ss_cworst_1p08_m40_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ss_cworst_1p08_m40_nldm.lib	2026-07-31 19:37:43.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ss_cworst_1p08_m40_nldm.lib	2026-08-21 14:20:01.773637116 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ss_rcworst_1p08_125_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ss_rcworst_1p08_125_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ss_rcworst_1p08_125_nldm.lib	2026-07-31 19:37:43.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ss_rcworst_1p08_125_nldm.lib	2026-08-21 14:20:01.120647936 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -591027,2 +591054,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -591152,2 +591180,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -591207,2 +591236,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ss_rcworst_1p2_m40_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ss_rcworst_1p2_m40_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ss_rcworst_1p2_m40_nldm.lib	2026-07-31 19:37:44.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_ss_rcworst_1p2_m40_nldm.lib	2026-08-21 14:20:01.330644456 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -591027,2 +591054,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -591152,2 +591180,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -591207,2 +591236,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
diff --color -U 1 -rN ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_typ_tt_1p2_25_nldm.lib IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_typ_tt_1p2_25_nldm.lib
--- ../icsprout55-old/IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_typ_tt_1p2_25_nldm.lib	2026-07-31 19:37:45.000000000 +0800
+++ IP/STD_cell/ics55_LLSC_H7C_V1p10C100/ics55_LLSC_H7CR/liberty/ics55_LLSC_H7CR_typ_tt_1p2_25_nldm.lib	2026-08-21 14:20:02.277628777 +0800
@@ -587733,2 +587733,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -587858,2 +587859,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -587913,2 +587915,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588099,2 +588102,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588224,2 +588228,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588279,2 +588284,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588465,2 +588471,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588590,2 +588597,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -588645,2 +588653,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -588831,2 +588840,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -588956,2 +588966,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589011,2 +589022,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589197,2 +589209,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589322,2 +589335,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589377,2 +589391,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589563,2 +589578,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -589688,2 +589704,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -589743,2 +589760,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -589929,2 +589947,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590054,2 +590073,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590109,2 +590129,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590295,2 +590316,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590420,2 +590442,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590475,2 +590498,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -590661,2 +590685,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -590786,2 +590811,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -590841,2 +590867,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
@@ -591027,2 +591054,3 @@
       direction : output;
+      clock_gate_out_pin : true;
       output_voltage : default_VDD_VSS_output;
@@ -591152,2 +591180,3 @@
       clock : true;
+      clock_gate_clock_pin : true;
       direction : input;
@@ -591207,2 +591236,3 @@
       direction : input;
+      clock_gate_enable_pin : true;
       driver_waveform_fall : "PreDriver20.5:fall";
